BioscoopSysteemAPI/BioscoopSysteemAPI/Migrations/CinemaDbContextModelSnapshot.cs

```diff
@@ -61,32 +64,67 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         new
                         {
                             MovieId = 1,
+                            Add3DMovie = true,
                             AllowedAge = (byte)16,
+                            Date = new DateTime(2023, 3, 9, 16, 18, 4, 180, DateTimeKind.Local).AddTicks(2502),
                             Description = "Lorem ipsum dolor sit amet",
                             ImageUrl = "/Images/Movies/Movie1.jpeg",
                             Name = "ScaryMovie",
-                            Price = 12,
-                            RoomId = 5
+                            Price = 12
                         },
                         new
                         {
                             MovieId = 2,
+                            Add3DMovie = false,
                             AllowedAge = (byte)8,
+                            Date = new DateTime(2023, 3, 9, 16, 18, 4, 180, DateTimeKind.Local).AddTicks(2543),
                             Description = "Lorem ipsum dolor sit amet",
                             ImageUrl = "/Images/Movies/Movie2.jpeg",
                             Name = "AntMan",
-                            Price = 9,
-                            RoomId = 4
```

Comment:
Klopt het dat de zaal weg is uit de movie tabel? Een film moet toch gekoppeld zijn aan een zaal. Of heb je dit nu op een andere manier werkend gemaakt?

Reply:
Ja dat klopt. Nu heb je de MovieRoom koppel tabel

Reply:
Okay, thanks voor de toelichting